src/Microsoft.Framework.Localization/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Microsoft.Framework.Localization.Test")]
```

Comment:
Not sure what @davidfowl thinks, but in some of the projects, we moved Internal classes to an .Internal namespace and made them public for testability instead. We don't seem to be following this on all projects, so just something to think about.